README.md

```diff
@@ -1,43 +1,59 @@
+| ------ | ----------- |
+|[![WhyCon applications](https://raw.githubusercontent.com/wiki/gestom/WhyCon/pics/whycon.png)](https://www.youtube.com/watch?v=KgKrN8_EmUA"AAAA")|-precise docking to a charging station (EU project STRANDS),<br/> -fitness evaluation for self-evolving robots (EU proj. SYMBRION),<br/>-relative localization of UAV-UGV formations (CZ-USA project COLOS),<br/>-energy source localization in (EU proj REPLICATOR),<br/>-robotic swarm localization (EU proj HAZCEPT).|
 
-WhyCon is proposed as an alternative to widely used and expensive localization systems. It is fully open-source.
+####Who created it ?
+The _WhyCon_ system was developed as a joint project between the University of Buenos Aires, Czech Technical University and University of Lincoln, UK.
+The main contributors were [Matias Nitsche](https://scholar.google.co.uk/citations?user=Z0hQoRUAAAAJ&hl=en&oi=ao), [Tom Krajnik](http://scholar.google.co.uk/citations?user=Qv3nqgsAAAAJ&hl=en&oi=ao) and [Jan Faigl](https://scholar.google.co.uk/citations?user=-finD_sAAAAJ&hl=en).
 
-# Publication and citation
+####Any details ?
 
-If you use this software for your publication it is mandatory to cite using the references in the provided `cite.bib`
-file (you can download it [here](https://raw.github.com/lrse/whycon/master/cite.bib)).
+A six-page overview of the <i>WhyCon</i> system was first presented at the ICAR conference [[1](#references)].
+A deteiled description was published in the Journal of Intelligent and Robotics Systems [[2](#references)].
+An overview of _WhyCon's_ applications was presented at the Workshop on Open Source Aerial Robotics during the International Conference on Intelligent Robotic Systems, 2015 [[3](#references)].
+Nowadays, the system is being used in several research projects the globe, see the following video for examples.
 
-Note that this .bib includes not only the references to the scientific works that describe the underlying method, but
-also a reference to the implementation for a specific (stable) version of the code on GitHub (look for the DOI containing
-the word "zenodo").
+####How can I use it ?
+ 
+If you decide to use this _WhyCon_ for your research, please cite it using the one of the references provided in this [bibtex](http://raw.githubusercontent.com/wiki/gestom/WhyCon/papers/WhyCon.bib) file.
+Note that this .bib includes not only the references to the scientific works that describe the underlying method, but also a reference to the implementation for a specific (stable) version of the code on GitHub (look for the DOI containing the word "zenodo").
 
-# Installation
+------
+###References
+1. T. Krajník, M. Nitsche et al.: <b>[External localization system for mobile robotics.](http://raw.githubusercontent.com/wiki/gestom/WhyCon/papers/2013_icar_whycon.pdf)</b> International Conference on Advanced Robotics (ICAR), 2013. [[bibtex](http://raw.githubusercontent.com/wiki/gestom/WhyCon/papers/2013_icar_whycon.bib)].
```

Comment:
there's already a cite.bib file on the repo: https://github.com/lrse/whycon/blob/master/cite.bib
I think it would be better to link to that. You can update that file on your fork and include the change in this pull request if you like. You can also add a "papers" directory on the repo and add the PDFs there.